x-pack/plugins/cases/server/routes/api/__fixtures__/route_contexts.ts

```diff
@@ -56,18 +55,23 @@ export const createRouteContext = async (client: any, badAuth = false) => {
   contextMock.core.savedObjects.getClient = jest.fn(() => client);
   contextMock.core.savedObjects.client = client;
 
+  // create a fake feature
+  const featureStart = featuresPluginMock.createStart();
+  featureStart.getKibanaFeatures.mockReturnValue([
+    // all the authorization class cares about is the `cases` field in the kibana feature so just cast it to that
+    ({ cases: ['securitySolution'] } as unknown) as KibanaFeature,
+  ]);
+
   const factory = new CasesClientFactory(log);
   factory.initialize({
     alertsService,
     caseConfigureService,
     caseService,
     connectorMappingsService,
     userActionService,
-    featuresPluginStart: featuresPluginMock.createStart(),
+    featuresPluginStart: featureStart,
     getSpace: async (req: KibanaRequest) => undefined,
-    isAuthEnabled: false,
-    securityPluginSetup: securityMock.createSetup(),
-    securityPluginStart: securityMock.createStart(),
+    // intentionally not passing the security plugin so that security will be disabled
```

Comment:
So our unit test will not test the authorization as our integration tests do that, right?

Reply:
Good point. I'll try writing some just so we have coverage on both. I can't remember if I ran into issues with the features related checks. I'll let you know 👍